scripts/distribution/builder.Dockerfile

```
@@ -78,7 +78,7 @@ ENV ENABLE_TERM_HANDLER=true
 
 RUN apt-get update && apt-get install -y unbound curl netbase ca-certificates supervisor nginx libnuma1 libtinfo6 libpq-dev liblmdb-dev jq
 RUN curl -L https://getenvoy.io/cli | bash -s -- -b /usr/local/bin
```

Possibly we should change this to:
```suggestion
RUN curl -L https://getenvoy.io/install.sh | bash -s -- -b /usr/local/bin
```
in line with the current getenvoy documentation. In any case, both currently redirect to the same URL, but who knows if they'll decide to change that.

---

Yes I agree. I think we should look in other ways of acquiring the getenvoy.